containers/unit_tests/TestErrorReporter.hpp

```diff
@@ -234,7 +235,9 @@ struct ErrorReporterDriverNativeOpenMP
 };
 #endif
 
-#if defined(KOKKOS_CLASS_LAMBDA) && \
+// FIXME_MSVC MSVC just gets confused when using the base class in the
+// KOKKOS_CLASS_LAMBDA
+#if !defined(KOKKOS_COMPILER_MSVC) && defined(KOKKOS_CLASS_LAMBDA) && \
```

Comment:
Isn't `KOKKOS_CLASS_LAMDA` always defined now?

Reply:
Yes, it is, but I avoided doing all the cleanup in this pull request.